src/Sarif/CodeGenHints.json

```diff
@@ -63,6 +83,26 @@
       }
     }
   ],
+  "exceptionData": [
+    {
+      "kind": "BaseTypeHint",
+      "arguments": {
+        "baseTypeNames": [
+          "PropertyBagHolder"
+        ]
+      }
+    }
+  ],
+  "externalFiles": [
```

Comment:
>externalFiles [](start = 3, length = 13)

This doesn't look right. The `run` object has a property `externalFiles`, which is of type "array of `externalFile`, and the `externalFile` object is the thing that has a property bag.

Reply:
Sorry, I got this wrong. The spec says that the value of `run.externalFiles` is an object, and then it goes on to describe the properties of that object -- but it doesn't give that set of properties a name, unlike every other object in the spec. I'll file an editorial issue to fix that.

---
In reply to: [226708519](https://github.com/Microsoft/sarif-sdk/pull/1114#discussion_r226708519) [](ancestors = 226708519)